### DISCUSSION TITLE
Not showing detail usage of CPU and memory

### DISCUSSION THREAD

**Discussion by buiducnhat** [cockpit-project/cockpit#18861] [Q&A]
Here is my overview tab of cockpit dashboard:

![](https://uphinh.vn/images/2023/05/30/c2c4d9f416696d8bfaeaf1a5ad7a14f3.png)

It doesn't show the detail usage of CPU and Memory (0% and availble 3.8gi (100%)) after I install metrics chart like below.
Can I fix this, thank you!

**Comment by jelly** [comment]
What version of Cockpit and on what sort of hardware?

**Reply by buiducnhat** [reply]
I install the latest of Cockpit on Github, this is my hardware infor in neofetch
<img width="820" alt="Screenshot 2023-05-31 at 22 05 47" src="https://github.com/cockpit-project/cockpit/assets/59013407/8a3104f6-9e9c-48d7-85aa-45a331d0dd6a">

**Comment by jelly** [comment]
Is there anything in the browser developer console?

**Comment by jrkagan** [comment]
I am having the same exact issue on a very different configuration. Have tried multiple browsers. CPU and memory both stuck at static 0% usage. Did anyone ever find a resolution for this?

**Reply by martinpitt** [reply]
This hasn't even been diagnosed yet. We still need more information, like at least the browser console messages. Are you also running a custom built `make install`, or some official Linux distro packages?

**Reply by jrkagan** [reply]
Thanks. I don't see anything relevant in the browsers' consoles. I'm running the official backports version on 22.04.3 LTS on a Rasp Pi.
